app/renderer/components/downloadItem.js

```diff
@@ -73,6 +73,8 @@ class DownloadItem extends ImmutableComponent {
   }
   render () {
     const origin = getOrigin(this.props.download.get('url'))
+    const localFileOrigins = ['file:', 'blob:', 'data:', 'chrome-extension:', 'chrome:']
+    const localFile = localFileOrigins.some((localFileOrigin) => origin.startsWith(localFileOrigin))
```

Comment:
Did you mean like this:  origin && localFileOrigins.some((localFileOrigin) => origin.startsWith(localFileOrigin)) @diracdeltas